narwhal/primary/src/synchronizer.rs

```diff
@@ -45,15 +44,17 @@ pub struct Synchronizer {
 impl Synchronizer {
     pub fn new(
         name: PublicKey,
-        committee: SharedCommittee,
+        _committee: SharedCommittee,
         worker_cache: SharedWorkerCache,
         certificate_store: CertificateStore,
         payload_store: Store<(BatchDigest, WorkerId), PayloadToken>,
         tx_certificate_fetcher: mpsc::Sender<Certificate>,
         rx_consensus_round_updates: watch::Receiver<Round>,
         dag: Option<Arc<Dag>>,
+        genesis_certs: Vec<Certificate>,
```

Comment:
In backports, there's now a `genesis: HashMap<CertificateDigest, Certificate>` here. Maybe we can use that for our genesis certs without having to additionally pass a `Vec<Certificate>`

Reply:
that seems ok to me